developer/src/kmcmpdll/keyman64.h

```diff
@@ -30,7 +30,7 @@
 #include <windows.h>
 #include <assert.h>
 
-#include "compiler.h"
+#include "../../../../common/windows/cpp/include/legacy_kmx_file.h"
```

Comment:
I think that's one `../` too much

Reply:
You're right, but weirdly enough it builds?!  If I add an extra `../` it fails to build. But it builds when it has one too many `../` components... Is there some bizarre pathing standard I am not aware of here?

Anyway, I have corrected it!